### ISSUE TITLE
RequestPolicy freezes up theaustralian.com.au

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#279]
```
imported trac ticket
created: 2011-11-25 02:29:40
reporter: intgr
```

Whenever I navigate to the theaustralian.com.au website with RequestPolicy enabled, Firefox freezes entirely.

I'm using Firefox 8.0.1 on 64-bit Arch Linux. Is reproducible even when I disable all other addons.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-12-20 15:39:52
author: justin
```

My sincere apologies for the delay in replying to and addressing this bug. Being a student, the time near the end of the semester can really get in the way of important things like triaging and fixing crash/freeze bugs. (!)

I've confirmed the bug and will be fixing it soon.

**Comment by jsamuel** [comment]
Fixed by 12beed0b980cebe45e4ca82b895e3f4b05de4456. Beta version 0.5.24b1 ([download](https://www.requestpolicy.com/releases/requestpolicy-0.5.24b1.xpi)) includes this fix.
